### DISCUSSION TITLE
Design + Marketing + layout

### DISCUSSION THREAD

**Discussion by guimit** [BrasilAPI/BrasilAPI#240] [Ideas]
Olá amigos, estive a ver sobre este projeto no YouTube e acredito muito no potencial do mesmo. Sou designer e tenho já 5 anos focados só em UX/UI e gostaria muito de contribuir a este projeto.

Os tópicos iniciais que eu gostaria de discutir com vocês são:

**Sobre a marca:**  Já foi feita uma estratégia de branding para ela?
**Sobre o site:** Como o site vai entregar a informação aos utilizadores? Vamos apresentar uma wiki ou vamos criar algo que seja fora do padrão? Algo que surpreenda nossa equipe?
**Leads** Como vamos nos relacionar com os com as pessoas que irão utilizar a plataforma? Vamos trabalhar alguma estratégia de nutrição de leads para fortalecer a comunidade?

Para ser bem sincero tem muito mais perguntas, mas vamos por partes. Aguardo o vosso retorno.

**Comment by gasampaiosouza** [comment]
Acredito que por ser uma iniciativa "nova" focada mais na parte da API, eles não tem tanta preocupação com isso. Mas podemos ir pensando sobre algo assim com o tempo. Também estou pensando em fazer algum layout com o estilo básico de sites de API, colocando uma sidebar com os tópicos, toda a documentação com um layout fixo, etc. Podemos ir conversando mais sobre isso!

**Comment by RodriAndreotti** [comment]
Cara, com relação a layout para a documentação nem precisa desenvolver muita coisa, tem alguma ferramentas prontas, como o Swagger UI, eu utilizo uma que usa as especificações da openapi (Do swagger, por consequência) chamada Redoc (https://redocly.github.io/redoc/), seria interessante mesmo um layout para o site.

**Comment by patrickdeangelis** [comment]
Acho uma ótima ideia disponibilizar a documentação com openapi e Redoc, jogava em um docs.brasilapi.com.br, ficaria bem legal e poderíamos já utilizar a documentação como forma de SEO para o produto(Brasil API).

**Comment by RodriAndreotti** [comment]
Bem lembrado o uso do subdomínio e o SEO, com certeza dará mais visibilidade ao projeto.
Fora que o uso de uma lib opensource acho que ficaria mais alinhado a proposta do projeto.

**Comment by RodriAndreotti** [comment]
Pessoal, com relação a documentação, vale lembrar que já tem um PR sobre aqui #101 , incluso a pessoa que fez o PR acatou a sugestão do ReDocs ;-)

**Comment by enieber** [comment]
só para linkar com essa issue, #93
